helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -411,6 +412,12 @@ internal fun OrtResult.getUnresolvedRuleViolations(): List<RuleViolation> {
     }
 }
 
+/**
+ * Return a copy with the [ErrorResolution]s replaced by the given [errorResolutions].
+ */
+internal fun RepositoryConfiguration.replaceErrorResolutions(errorResolutions: List<ErrorResolution>):
+        RepositoryConfiguration = copy(resolutions = (resolutions ?: Resolutions()).copy(errors = errorResolutions))
```

Comment:
The return type should be in the same line as the rest of the signature (as in the functions below).

Reply:
Done.